apps/frontend/src/app/2_molecules/ConnectWalletButton/ConnectWalletButton.tsx

```diff
@@ -39,6 +41,14 @@ export const ConnectWalletButton: FC<
         address={address}
         dataLayoutId={dataLayoutId}
         className={className}
+        balance={Object.values(Asset).map(asset => (
+          <SimpleTableRow
+            key={asset}
+            className="text-sov-white"
+            label={asset}
+            value="0"
```

Comment:
Once you have the new component I mentioned above, you need to fetch the balances there. @soulBit created a new story for it: https://sovryn.atlassian.net/browse/SOV-1101 so you don't need to add it in this PR.